run_benchmark.py

```diff
@@ -625,15 +685,25 @@ def load_or_featurize(task):
         raise RuntimeError(f"No folder found for {task!r}.")
 
     os.chdir(task)
-
-    setup_parallelism(4, 4)
+    #setup_parallelism(4,4)
```

Comment:
As with the comment above, I think it is clearer to do `setup_parallelism(1, 1)` to indicate that there is no thread-based parallelism. We should probably even rename the function altogether

```suggestion
    setup_parallelism(1, 1)
```